duckduckgo/macos-browser
# Fix an issue with NetP debug menu options

Task/Issue URL: https://app.asana.com/0/1199230911884351/1205653149658149/f
Tech Design URL:
CC: @diegoreymendez 

**Description**:

This PR fixes a recent issue with the NetP debug menu options.

This issue only affects the Privacy Pro target.

**Steps to test this PR**:
1. Launch the main target, make sure the NetP feature flag override settings work
2. Launch the DuckDuckGo Privacy Pro target, make sure the NetP feature flag override settings work



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
